api/groups.go

```diff
@@ -24,6 +26,7 @@ const (
 
 // CreateGroupHandler creates a new group
 func (api *API) CreateGroupHandler(ctx context.Context, w http.ResponseWriter, req *http.Request) (*models.SuccessResponse, *models.ErrorResponse) {
+	sortBy := req.URL.Query().Get("sortBy")
```

Comment:
can you coordinate with @dionisislolas  about what parameter to use please (`sortBy` or `sort`)